### DISCUSSION TITLE
Read Parquet timestamp with time zone type columns as time type without time zone?

### DISCUSSION THREAD

**Discussion by ghost** [duckdb/duckdb#5915] [Q&A] [answered]
When reading Parquet files in DuckDB (0.6.1), I noticed that the timestamp with time zone type columns are now just timestamp type.
Is this perhaps a bug? Or is this intentional behavior?

I am confused because other libraries (pyarrow, py polars, R arrow) recognize the time zone.

**Comment by Mytherin** [comment, marked as answer]
Could you provide an example? That does not seem intentional.

**Reply by ghost** [reply]
Thanks for your response!
Upload a sample file. [sample.zip](https://github.com/duckdb/duckdb/files/10423277/sample.zip)

```sh
❯ duckdb -c "from sample.parquet"
┌─────────────────────┐
│    TimeRecStart     │
│      timestamp      │
├─────────────────────┤
│ 2022-10-17 21:52:27 │
│ 2022-10-17 21:52:27 │
└─────────────────────┘

❯ python -c 'import polars as pl; df=pl.read_parquet("sample.parquet"); print(df)'
shape: (2, 1)
┌─────────────────────────┐
│ TimeRecStart            │
│ ---                     │
│ datetime[ms, UTC]       │
╞═════════════════════════╡
│ 2022-10-17 21:52:27 UTC │
│ 2022-10-17 21:52:27 UTC │
└─────────────────────────┘

❯ R.exe -s -e 'arrow::read_parquet("sample.parquet", as_data_frame = FALSE)'
Table
2 rows x 1 columns
$TimeRecStart <timestamp[ms, tz=UTC]>
```

**Reply by Mytherin** [reply]
Thanks! I have pushed a PR that should fix this issue - see #5916

**Reply by ghost** [reply]
Awesome! Thank you for your very quick update!
